scripts/west_commands/sbom/README.rst

```diff
@@ -0,0 +1,370 @@
+     .. parsed-literal::
+        :class: highlight
+
+        west ncs-sbom --input-files *files ...* --license-detectors scancode-toolkit --output-cache-database *cache-database.json*
+
+If you prefer a non-default set of detectors, you can provide a list of comma-separated detectors with the ``--license-detectors`` option, for example:
+
+  .. code-block:: bash
+
+     --license-detectors spdx-tag,scancode-toolkit
+
+Some of the detectors are optional, which means that they are not executed for a file that
+already has licenses detected by some other previously executed detector.
+Detectors are executed from left to right using a list provided by the ``--license-detectors``.
+
+  .. code-block:: bash
+
+     --optional-license-detectors scancode-toolkit
+
+Some detectors may run in parallel on all available CPU cores, which speeds up the detection time.
+Use the ``-n`` option to limit the number of parallel threads or processes.
+
+.. _west_sbom HTML report overview:
+
+HTML report overview
+********************
+
+The HTML report has following structure:
+
+* Summary of the report, containing the following:
+
+   * Notes at the beginning.
+
+     General information on the report.
+   * List of inputs.
+
+     The file sources.
+   * List of licenses.
+
+     All licenses detected in the input files.
+   * List of added license texts.
+
+     If a license is not in the `SPDX License List`_ and it is in the internal database,
+     the license text is added to the report.
+
+  You can click links in the summary to get more details about specific items.
+
+* List of files without any license information or with license information that cannot be detected automatically.
+
+  You have to investigate them manually to get the license information.
+
+* Details about each detected license:
+
+   * License identifier.
+   * Information if it is a standard SPDX license.
+   * License name if available.
+   * Link to license text or more details if available.
+   * All files from the input covered by this license.
+
+* License texts added to this report.
+
+.. _west_sbom Extracting from build:
+
+Extracting a list of files from a build directory
+*************************************************
+
+The ``ncs-sbom`` extracts a list of files from a build directory.
+It queries ``ninja`` for the targets and dependencies.
+
+The entry point is the :file:`zephyr/zephyr.elf` target file.
+The script asks ``ninja`` for all input targets of the :file:`zephyr/zephyr.elf` target.
+It also asks for all input targets of the previously extracted input targets,
+until it reaches all leaves in the dependency tree.
+The result is a list of all the leaves.
+
+To change the target or specify multiple targets, you can add them after the build directory in the ``-d`` option, for example:
+
+.. parsed-literal::
+   :class: highlight
+
+   -d build_directory *target1.elf* *target2.elf*
+
+There are two redundant methods for improving the correctness of the above algorithm:
```

Comment:
```suggestion
There are two alternative methods for improving the correctness of the above algorithm:
```

Reply:
In this context `redundancy` does not mean the same as `alternative`.

Reply:
Okay. How about "additional"?